components/myforms/Tabs/Tab.tsx

```diff
@@ -32,6 +34,7 @@ export const Tab = (props: TabProps): React.ReactElement => {
       aria-selected={isActive ? "true" : "false"}
       aria-controls={tabpanelId}
       onClick={callback}
+      {...(!isActive && { tabIndex: "-1" })}
```

Comment:
So the tabIndex and aria-selected is controlled here. Then in the vanilla TabsKeynav the current element focus and other behavior is controlled. These could easily be moved into the TabsKeynav class if this separation is a bit confusing